### DISCUSSION TITLE
Stuck at "addDoubletScores"

### DISCUSSION THREAD

**Discussion by baijiexu** [GreenleafLab/ArchR#1646] [Questions / Documentation]
Hi,

When I run the addDoubletScores, it always stuck there. Even I let it go for one night, there is no progress. And there is no error at all. I have no idea. I ran it using `proj_Sim1Cre <- addDoubletScores(proj_Sim1Cre)`

Here is the log file.
[ArchR-addDoubletScores-5a4e42f1cbef-Date-2022-09-27_Time-10-00-51.log](https://github.com/GreenleafLab/ArchR/files/9657396/ArchR-addDoubletScores-5a4e42f1cbef-Date-2022-09-27_Time-10-00-51.log)

Thank you so much.
BJ

**Comment by rcorces** [comment]
I unfortunately dont have any insight here since there is no error or relevant info in the log. I assume it works on the tutorial data?

**Reply by baijiexu** [reply]
No. It doesn't work neither. I am really confused about it.
[ArchR-addDoubletScores-a6ad20ad9fb5-Date-2022-09-27_Time-19-17-44.log](https://github.com/GreenleafLab/ArchR/files/9665619/ArchR-addDoubletScores-a6ad20ad9fb5-Date-2022-09-27_Time-19-17-44.log)

**Reply by rcorces** [reply]
If it doesnt work on the tutorial data, that is very suspicious. The tutorial has been run by hundreds (maybe thousands) of users at this point. This would mean that it is likely something specific to your computational environment that is causing problems. I would try this on a different machine.

**Reply by baijiexu** [reply]
I tried to reinstall the package in our system. But it still doesn't work. It's really weird. I really don't know what's the problem.

**Reply by baijiexu** [reply]
Hi Ryan, 
Thank you so much for your help. I figured it out. It is a parallelization problem in our system. After changing threads to 1, everything is great now.
Have a good day. BJ
